types/log/index.d.ts

```diff
@@ -0,0 +1,61 @@
+// Type definitions for log 6.3
+// Project: https://github.com/medikoo/log#readme
+// Definitions by: Adam Jones <https://github.com/domdomegg>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/log/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me